lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1929,25 +1941,32 @@ class BuildParGroups : public calyx::FuncOpPartialLoweringPattern {
     SmallVector<int64_t> indices = lbVals;
 
     while (true) {
+      // Create an `scf.execute_region` to wrap each unrolled block since
```

Comment:
Should this comment be placed where the execute region is created?

Reply:
sounds good